### PR TITLE
Add batch_norm、dropout and transpose2

### PR DESCRIPTION
**Pull request by yeliang2258**
1. Add batch_norm、dropout and transpose2
2. mv scale from nn to tensor

### CODE REVIEW

File: tests/test_auto_scan_dropout.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            axis = None
+        else:
+            axis = draw(
+                st.integers(
+                    min_value=0, max_value=len(input_shape) - 1))
+
+        config = {
+            "op_names": ["dropout"],
+            "test_data_shapes": [input_shape],
+            "test_data_types": [[dtype]],
```
Comment (jiangjiajun) [inline tests/test_auto_scan_dropout.py:71]: dropout支持float32和64，添加下单测

File: tests/onnxbase.py [hunk 2]
```diff
@@ -131,6 +131,8 @@ def __init__(self,
         else:
             self.places = ['cpu']
         self.name = file_name
+        if self.name == '':
+            self.name = "no_name"
```
Comment (jiangjiajun) [inline tests/onnxbase.py:135]: 单测的OP，从最小版本一直测到15

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: tests/onnxbase.py (hunk 2 above)
Comment (yeliang2258) [reply tests/onnxbase.py:135]: done

File: tests/test_auto_scan_dropout.py (hunk 1 above)
Comment (yeliang2258) [reply tests/test_auto_scan_dropout.py:71]: done

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**